src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -431,6 +431,23 @@
              Description="The User Pool ID" />
   </Provider>
 
+  <!--
+                                ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                ██ ▄▄▀██ ▄▄▄ ██ ▀██ █▄▄ ▄▄██ ▄▄▄██ ▀██ █▄▄ ▄▄██ ▄▄▄██ ██ ██ █████
+                                ██ █████ ███ ██ █ █ ███ ████ ▄▄▄██ █ █ ███ ████ ▄▄███ ██ ██ █████
+                                ██ ▀▀▄██ ▀▀▀ ██ ██▄ ███ ████ ▀▀▀██ ██▄ ███ ████ █████▄▀▀▄██ ▀▀ ██
+                                ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <Provider Name="Contentful" DisplayName="Contentful" Id="3d820b1e-33c7-4934-a783-48745343d53b"
```

Comment:
```suggestion
  <Provider Name="Contentful" Id="3d820b1e-33c7-4934-a783-48745343d53b"
```

Tip: when the display name matches the provider name, no need to set it explicitly 😃